File: blogmgr-core/src/main/java/com/huangtl/blogmgr/model/blog/SystemMemo.java
package com.huangtl.blogmgr.model.blog;

import com.huangtl.blogmgr.model.common.ObjectValue;

/**
 * 系统备忘录
 * @author Canaan
 * @date 2017年4月29日
 *
 */
public class SystemMemo extends ObjectValue {
	private static final long serialVersionUID = 1036876574138706523L;
	private String fId;
	private String fParentId;	//父id
	private String fHierarchy;	//层级关系
	private Integer fOrder;		//排序
	private String fTitle;		//标题
	private String fDescr;		//描述
	private String fUrl;		//连接
	public String getfId() {
		return fId;
	}
	public void setfId(String fId) {
		this.fId = fId;
	}
	public String getfTitle() {
		return fTitle;
	}
	public void setfTitle(String fTitle) {
		this.fTitle = fTitle;
	}
	public String getfDescr() {
		return fDescr;
	}
	public void setfDescr(String fDescr) {
		this.fDescr = fDescr;
	}
	public String getfUrl() {
		return fUrl;
	}
	public void setfUrl(String fUrl) {
		this.fUrl = fUrl;
	}
	public String getfParentId() {
		return fParentId;
	}
	public void setfParentId(String fParentId) {
		this.fParentId = fParentId;
	}
	public String getfHierarchy() {
		return fHierarchy;
	}
	public void setfHierarchy(String fHierarchy) {
		this.fHierarchy = fHierarchy;
	}
	public Integer getfOrder() {
		return fOrder;
	}
	public void setfOrder(Integer fOrder) {
		this.fOrder = fOrder;
	}
	
}

File: blogmgr/src/main/webapp/app/model/debug/SystemMemo.js
/**
 * 系统备忘录
 */
Ext.define('BlogMgr.model.debug.SystemMemo',{
	extend:'Ext.data.Model',
	idProperty:'fId',	//当前用户的主键标识
	fields:[
		{name:'fId',type:'string'},
		{name:'fParentId',type:'string'},
		//{name:'fHierarchy',type:'string'},
		{name:'fOrder',type:'int'},
		{name:'fTitle',type:'string'},
		{name:'fDescr',type:'string'},
		{name:'fUrl',type:'string'},
	],
	validators: {
		fId: 'presence',
		fTitle: 'presence'
	},
	proxy:{
		type:'ajax',
		url:'/blogmgr/debug/memo/get.data'
	}
});